src/agent/src/config.rs

```diff
@@ -175,6 +177,7 @@ pub struct AgentConfigBuilder {
     pub passfd_listener_port: Option<i32>,
     pub unified_cgroup_hierarchy: Option<bool>,
     pub tracing: Option<bool>,
+    pub expose_configfs: Option<bool>,
```

Comment:
In my glance of the item of `expose_configfs: Option<bool>`  here, I am confused, because I have it in mind with the previous definition of type `expose_configfs:  bool` in `struct  AgentConfig`.

Reply:
oops, my bad. :-)